cardano-tracer/src/Cardano/Tracer/Handlers/Metrics/Utils.hs

```diff
@@ -1,63 +1,42 @@
 {-# LANGUAGE NamedFieldPuns #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ViewPatterns #-}
 
 module Cardano.Tracer.Handlers.Metrics.Utils
-  ( MetricName
-  , MetricValue
-  , MetricsList
-  , RouteDictionary(..)
-  , getListOfMetrics
-  , renderListOfConnectedNodes
-  , renderJson
-  , nodeNames
-  , computeRoutes
+  ( module Cardano.Tracer.Handlers.Metrics.Utils
```

Comment:
I have never seen this! What is the meaning of this?

Reply:
It re-exports: https://stackoverflow.com/questions/36044041/haskell-module-inside-module

Reply:
This exports all symbols from that module. It's best to use the current module name here when explicit export lists are required, and you want to export *everything*.